tests/testthat/_snaps/dummy.md

```diff
@@ -25,47 +25,54 @@
       factors <- prep(factors, training = sacr_missing)
     Condition
       Warning:
-      ! There are new levels in a factor: `NA`.
+      ! There are new levels in `city`: NA.
+      i Consider using `recipes::step_unknown()` before `step_dummy()` to handle missing values.
```

Comment:
Feels a bit strange that one of these steps is namespaced and the other isn't.

Reply:
I'm doing this because `step_dummy()` is the function throwing the error. So it feels weird to link to it. Whereas `step_unknown()` might be unknown to the user

Reply:
It's possible that the user used `recipes::step_dummy()` rather than `step_dummy()` though, right?

I do think these should be consistent

Reply:
I see your point. was doing this to get the nice linking behavior 

![Screenshot 2024-05-30 at 11 27 06 AM](https://github.com/tidymodels/recipes/assets/14034784/a15f15f4-cb73-4ec4-baf7-1d18afbf907d)